mappings/net/minecraft/network/packet/s2c/query/MetadataS2CPacket.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_aleeatzq net/minecraft/network/packet/s2c/query/QueryResponseS2CPacket
+CLASS net/minecraft/unmapped/C_aleeatzq net/minecraft/network/packet/s2c/query/MetadataS2CPacket
```

Comment:
Just `Metadata` seems a little confusing. How about `QueryMetadata` or `ServerMetadata`?

Reply:
`ServerMetadata` works better, given that this is a response and not a query!